src/Sylius/Bundle/ShopBundle/Resources/config/routing/cart.yml

```diff
@@ -18,6 +18,7 @@ sylius_shop_cart_save:
         _sylius:
             template: "@SyliusShop/Cart/summary.html.twig"
             redirect: sylius_shop_cart_summary
+            checkout_redirect: sylius_shop_checkout_start
```

Comment:
I would be for creating a new route for starting the checkout with a proper redirect instead of customizing `saveAction` on order controller. IMO that would be clearer without the need for extra `if` statements